app/views/layouts/application.html.erb

```diff
@@ -42,7 +42,7 @@
         </div>
         <div class="govuk-header__content">
           <%= link_to "GOV.UK Rails Boilerplate", "/", class: "govuk-header__link govuk-header__link--service-name" %>
-          <button type="button" role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+          <button type="button" role="govuk-button" data-module="govuk-button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
```

Comment:
`role=` should still be `button` as that's the [HTML role attribute](https://stackoverflow.com/a/18664038).

Reply:
🤦🏽‍♂️